### ISSUE TITLE
[py-tx] using StopNCIIAPI.fetch_hashes will lead to ValueError due to missing field in StopNCIICSPFeedbackValue

### ISSUE THREAD

**Issue by Gnomeek** [facebook/ThreatExchange#1211]
Hey guys, I just got am `ValueError` when fetching hashes using `StopNCIIAPI.fetch_hashes`, the traceback is:
```
Traceback (most recent call last):
  File "/Users/will/Documents/Codespace/sds_pdq/cronjob/fetch_stopncii.py", line 105, in <module>
    main(x_functions_key, x_subscription_key)
  File "/Users/will/Documents/Codespace/sds_pdq/cronjob/fetch_stopncii.py", line 91, in main
    for resp in client.fetch_hashes_iter():
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/threatexchange/exchanges/clients/stopncii/api.py", line 251, in fetch_hashes_iter
    result = self.fetch_hashes(
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/threatexchange/exchanges/clients/stopncii/api.py", line 228, in fetch_hashes
    return dacite.from_dict(
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 63, in from_dict
    value = _build_value(type_=field.type, data=transformed_value, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 90, in _build_value
    return _build_value_for_collection(collection=type_, data=data, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 140, in _build_value_for_collection
    return data_type(_build_value(type_=item_type, data=item, config=config) for item in data)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 140, in <genexpr>
    return data_type(_build_value(type_=item_type, data=item, config=config) for item in data)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 92, in _build_value
    return from_dict(data_class=type_, data=data, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 63, in from_dict
    value = _build_value(type_=field.type, data=transformed_value, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 90, in _build_value
    return _build_value_for_collection(collection=type_, data=data, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 140, in _build_value_for_collection
    return data_type(_build_value(type_=item_type, data=item, config=config) for item in data)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 140, in <genexpr>
    return data_type(_build_value(type_=item_type, data=item, config=config) for item in data)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 92, in _build_value
    return from_dict(data_class=type_, data=data, config=config)
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/core.py", line 60, in from_dict
    transformed_value = transform_value(
  File "/Users/will/Library/Caches/pypoetry/virtualenvs/pdq-cronjob-gH7F_-yJ-py3.8/lib/python3.8/site-packages/dacite/types.py", line 18, in transform_value
    value = target_type(value)
  File "/Users/will/.pyenv/versions/3.8.14/lib/python3.8/enum.py", line 339, in __call__
    return cls.__new__(cls, value)
  File "/Users/will/.pyenv/versions/3.8.14/lib/python3.8/enum.py", line 663, in __new__
    raise ve_exc
ValueError: 'Undefined' is not a valid StopNCIICSPFeedbackValue
```
Maybe it's better to add `Undefined` in `StopNCIICSPFeedbackValue`?

**Comment by Gnomeek** [comment]
#1212

**Comment by Dcallies** [comment]
Thanks for looking into this!

**Comment by Dcallies** [comment]
Your fix should be in 1.0.7! https://pypi.org/project/threatexchange/
